### ISSUE TITLE
Training script and hyperparameter configuration

### ISSUE THREAD

**Issue by DamonAtSjtu** [LaVieEnRoseSMZ/AutoBNN#4]
I am impressed by your result that the accuracy of a much smaller model is higher that that of the Uniform-4X model.  However, I have some problems now. I cannot reproduce the results in your article. I want to know if you can share me with your training script and hyperparameter configuration, especially the training script of Resnet18 on ImageNet. Looking forward to your reply.
